example/spec/redeployment/LambdaVersionOption.js

```diff
@@ -32,41 +21,40 @@ describe('When the useWorkflowLambdaVersions option is set to false the deployme
   const startDate = new Date();
 
   beforeAll(async () => {
-    try {
-      // ensure we're working with a fresh deployment
-      await redeploy(config);
+    await redeploy(config);
```

Comment:
Should we redeploy with retries in this file as well?

Reply:
Of the two options, I'd rather remove the redeploys from the other file.  I'm not what situation we'd run into where a redeploy would fail, and immediately retrying it would succeed.

That being said, this PR is only targeted at making sure that any files overwritten as part of redeploys (workflows.yml, config.yml, and a lambda function) are deleted after the test runs.

We should have a conversation about redeploys and how tests behave on failure, but I think that needs to be a bigger discussion.